includes/apple-exporter/builders/class-components.php

```diff
@@ -30,7 +30,67 @@ protected function build() {
 		// such as if a thumbnail was used from the body.
 		$components = array_merge( $this->meta_components(), $components );
 
-		return $this->group_body_components( $components );
+		$grouped_components = $this->group_body_components( $components );
+		$grouped_components = $this->componentize( $grouped_components );
+
+		return $grouped_components;
+	}
+
+	/**
+	 * Give an array of components in array format, transform
+	 * all that are needed into components
+	 *
+	 * @param array $components
+	 * @return array
+	 * @access private
+	 */
+	private function componentize( $components ) {
+		$heading_items = array();
+		$header_pos = 0;
+
+		foreach ( $components as $i => $component) {
+			if ( 'title' === $component['role'] || 'byline' === $component['role'] ) {
+				$header_pos = ( 'title' === $component['role'] ) ? $i : $header_pos;
+				$heading_items[] = $component;
+			}
+			if ( 'quote' === $component['role'] ) {
+				array_splice( $components, $i, 1, array( array(
+					'role' => 'container',
+					'layout' => array(
```

Comment:
These probably can move to the `set_layout` method in `includes/apple-exporter/components/class-quote.php`